src/layout/Group/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { SummaryGroupComponent } from 'src/components/summary/SummaryGroupComponent';
+import { GroupRenderer } from 'src/layout/Group/GroupRenderer';
+import { ContainerComponent } from 'src/layout/LayoutComponent';
+import type { PropsFromGenericComponent } from 'src/layout';
+import type { SummaryRendererProps } from 'src/layout/LayoutComponent';
+
+export class Group extends ContainerComponent<'Group'> {
+  directRender(): boolean {
+    return true;
+  }
+
+  render(props: PropsFromGenericComponent<'Group'>): JSX.Element | null {
+    return <GroupRenderer {...props} />;
+  }
+
+  renderSummary({ change, summaryNode, targetNode, overrides }: SummaryRendererProps<'Group'>): JSX.Element | null {
+    return (
+      <SummaryGroupComponent
+        {...change}
+        summaryNode={summaryNode}
+        targetNode={targetNode}
+        overrides={overrides}
+      />
+    );
+  }
+
+  renderSummaryBoilerplate(): boolean {
+    return false;
+  }
+
+  useDisplayData(): string {
```

Comment:
Not related to this PR, but should this method have been renamed to getDisplayData? Within the React ecosystem, I associate the prefix "use" with a react-hook that uses other hooks. By reading the code within this class it looks like a normal function.

But again, maybe some other classes that are also inherent in the FormComponent actually use hooks within the useDisplayData method. 

No change is required - these were just some thoughts while reading the code.

Reply:
> But again, maybe some other classes that are also inherent in the FormComponent actually use hooks within the useDisplayData method.

Spot on! 😉 "most" components fetch their form data in this step, which many of them get from redux, meaning these methods are usually using hooks to fetch the data they need. This is why I had to disable the hook rules for these files, since the eslint rules thinks these files contain class components (which they don't, but it kinda looks like it). I would agree that it's a bit funky, though, and I think we can move the target a bit further and find a way to work around this while keeping our code somewhat nice.

I added a `getFormData()` function to `LayoutNode`, since formdata is an input when resolving the node tree anyway, so `LayoutNode` can find data in data bindings. That still doesn't work for a `list` binding in, for example, the `FileUpload` component - so that has to figure out the list binding on its own. There's still a way to go here, in other words!

Reply:
Thanks for the clarification! Anyway, we are in the right direction to make the codebase more maintainable. Great work! 🚀 For now I'm resolving this comment and we can discuss this later on. Maybe we somewhere in the future can fetch data and handle states differently. :)